etl/__main__.py

```diff
@@ -101,6 +105,19 @@ def get_app_variant_description(app):
     return description
 
 
+def get_expiry_date(expiry, app_name):
```

Comment:
This would probably be better named `get_expiry_text`.

Could you move this to a separate file called `expiry.py` and write some quick tests for it? You can probably model off of https://github.com/mozilla/glean-dictionary/blob/main/etl_tests/test_looker.py

(we won't want to query the network API in the test, but I can help with that part)